test/unit/spec/nbextensions/bulkImportCell/tabs/xsvGenerator-spec.js

```diff
@@ -3,73 +3,60 @@ define([
     '/narrative/nbextensions/bulkImportCell/tabs/xsvGenerator',
     'common/props',
     'common/ui',
-], ($, XSVGenerator, Props, UI) => {
+    'json!/test/data/kb_uploadmethods.import_fastq_interleaved_as_reads_from_staging.spec.json',
+    'json!/test/data/kb_uploadmethods.import_fastq_noninterleaved_as_reads_from_staging.spec.json',
+    'json!/test/data/kb_uploadmethods.import_sra_as_reads_from_staging.spec.json',
+], (
```

Comment:
use these existing spec files (I didn't see them until after I had written the first set of tests...)

Reply:
😅